bin/ember-component-template-colocation-migrator

```diff
@@ -14,8 +14,19 @@ let options = {
 let parsed = nopt(options);
 let projectRoot = parsed['project-root'] || process.cwd();
 
+const args = process.argv.slice(2);
+
+// Allow passing the flag, -fs (flat) or -ns (nested), to specify component structure
```

Comment:
I gave the names `-fs` and `-ns` to match Ember CLI's aliases. I'm not strongly tied to these names, but think they are nice because they are short to type.

Reply:
👍